test/cases/smoke/test.js

```diff
@@ -1,5 +1,7 @@
-import { ok } from 'node:assert';
+import { ok, strictEqual } from 'node:assert';
 
-export function test (instance) {
+export function test (instance, { imports, exports }) {
+  strictEqual(imports.length, 0);
+  strictEqual(exports.length, 0);
```

Comment:
I am not sure I follow this test. We get the list of imports based on the output of `splice_bindings` call where we will have the engine imports.

Reply:
Yes you're right, we should have two separate imports lists here to test it. Or we could have a metadata flag on the import like `Engine | Guest | Both` say.